cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -96,18 +98,36 @@ public void run() {
    *     Otherwise, the specified version will be installed in the software repository without touching and IDE installation and used to run.
    * @param args the command-line arguments to run the tool.
    */
+  public void runTool(ProcessMode processMode, VersionIdentifier toolVersion, boolean existsEnvironmentContext, String... args) {
+
+    Path binaryPath;
+    binaryPath = Path.of(getBinaryName());
+
+    if (existsEnvironmentContext) {
+      ProcessContext pc = this.context.newProcess().errorHandling(ProcessErrorHandling.WARNING).executable(binaryPath).addArgs(args);
+
+      if (toolVersion == null) {
+        install(pc, true);
+      } else {
+        throw new UnsupportedOperationException("Not yet implemented!");
```

Comment:
Even though marked as outdated, this is still present.

Reply:
I removed the check toolVersion==null, and as far as I tested everything runs smoothly. I also tested gcviewer manually with this change and it seems to work. It gets installed and it also runs

Reply:
My suggestion was to entirely remove the `toolVersion` if it is not supported at all and is not used anywhere.
Since we later might need to reintroduce it, I set this to resolved.